runner/dl_checkpoints.sh

```diff
@@ -79,7 +79,42 @@ function download_all_models() {
 
     # Download live-video-to-video models.
     huggingface-cli download KBlueLeaf/kohaku-v2.1 --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
-    huggingface-cli download KwaiVGI/LivePortrait --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download stabilityai/sd-turbo --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download warmshao/FasterLivePortrait --local-dir models/FasterLivePortrait--checkpoints
+}
+
+function build_tensorrt_models() {
+    download_all_models
+
+    printf "\nBuilding TensorRT models...\n"
+
+    # Matrix of models and timesteps to compile StreamDiffusion TensorRT engines for.
+    MODELS="stabilityai/sd-turbo KBlueLeaf/kohaku-v2.1"
+    TIMESTEPS="3 4" # This is basically the supported sizes for the t_index_list
+    docker run --rm -it -v ./models:/models --gpus all \
+        livepeer/ai-runner:live-app-streamdiffusion \
```

Comment:
I see now, sorry I was copying this into a separate script so to avoid the `download_all_models` and I lost those variables. Instead of downloading all models, maybe we can just download the ones we're building tensorrt for?